src/runner.ts

```diff
@@ -38,14 +38,42 @@ export async function runCli(fn: Runner, options: DetectOptions = {}) {
   }
 }
 
+export async function getRunCommand(
```

Comment:
```suggestion
export async function getCommand(
```

Maybe that would be enough?

Reply:
That was my initial solution, but `getCommand` would conflict with the `getCommand` exported in `./parse`. I was also thinking about something like `getCLICommand`. WDYT?